goblint.opam.locked

```diff
@@ -105,6 +105,7 @@ depends: [
   "stdlib-shims" {= "0.3.0"}
   "stdune" {= "3.16.0"}
   "stringext" {= "1.6.0"}
+  "thread-table" {= "1.0.0"}
```

Comment:
Why is this needed?

Reply:
It's a transitive dependency of domain-local-await. I guess in #1708 the lock file was manually modified and this was forgotten.